app/views/responses/_export_csv_modal.html.erb

```diff
@@ -8,24 +8,23 @@
         </div>
         <div class="modal-body">
           <div class="fields">
-            <% if responses.total_entries >= ResponsesController::CSV_EXPORT_LIMIT %>
-              <div class="alert alert-danger">
-                <%= t("response.export_options.export_error",
-                  threshold: number_with_delimiter(ResponsesController::CSV_EXPORT_LIMIT)) %>
-              </div>
-            <% elsif responses.total_entries >= ResponsesController::CSV_EXPORT_WARNING %>
-              <div class="alert alert-warning">
-                <%= t("response.export_options.export_warning",
-                  threshold: number_with_delimiter(ResponsesController::CSV_EXPORT_WARNING)) %>
-              </div>
-            <% end %>
+            <!-- Shown via JS. -->
+            <div id="export-count-error" class="alert alert-danger hide">
+              <%= t("response.export_options.export_error",
+                threshold: number_with_delimiter(ResponsesController::CSV_EXPORT_LIMIT)) %>
```

Comment:
thanks for cleaning this up!

Reply:
Your version worked great btw, this only needed to change because now `responses.total_entries` can be dynamically overridden if the user checks the boxes.